src/core/utils/omit-deep.ts

```diff
@@ -1,9 +1,10 @@
 import type { GoslingSpec } from '@gosling.schema';
-import { cloneDeepWith, type CloneDeepWithCustomizer } from 'lodash';
+import { cloneDeepWith } from 'lodash-es';
+import { isObject } from '../gosling.schema.guards';
 
 export function omitDeep(spec: GoslingSpec, omitKeys: string[]) {
-    return cloneDeepWith(spec, (value: CloneDeepWithCustomizer<GoslingSpec>) => {
-        if (value && typeof value === 'object') {
+    return cloneDeepWith(spec, (value: unknown) => {
```

Comment:
I didn't know about this `unknown` type, seems quite useful!

Reply:
Yup, basically a [type-safe counterpart to `any`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html#new-unknown-top-type). Anything can be assigned to `unknown`, but then you must work to convince the compiler that you can access particular properties. There's been some discussion about changing the return type of `Response.json()` to `Promise<unknown>`.